web/src/Routes.js

```diff
@@ -10,7 +10,7 @@ const Routes = () => {
 
         <Private unauthenticated="home">
           <Route path="/onboard" page={OnboardPage} name="onboard" />
-          <Route path="/dashboard" page={DashboardPage} name="dashboard" />
+          <Route path="/setting" page={SettingPage} name="setting" />
```

Comment:
Can we change this route to `/settings`?

```suggestion
          <Route path="/settings" page={SettingsPage} name="settings" />
```